src/components/VolumeSlider/VolumeSlider.js

```diff
@@ -10,7 +10,60 @@ class VolumeSlider extends Component {
     onVolumeChanged: PropTypes.func.isRequired,
   };
   render() {
-    return <div>VolumeSlider</div>;
+    const { volume, onVolumeChanged } = this.props;
+    const width = 294;
+    const maxVolume = 100;
+    const currentVolumeWidth = (volume / maxVolume) * width;
+    return (
+      <div className="volume-slider">
+        <svg
+          version="1.0"
+          xmlns="http://www.w3.org/2000/svg"
+          xmlnsXlink="http://www.w3.org/1999/xlink"
+          x="0"
+          y="0"
+          viewBox={`0 0 ${width} 24`}
+          style={{
+            enableBackground: `new 0 0 ${width} 24`,
+          }}
+          xmlSpace="preserve"
+        >
+          <polygon
+            className="volume-slider__background"
+            points={`0,24 ${width},0 ${width},24`}
+          />
+          <g>
+            <defs>
+              <rect
+                id="CurrentVolumeMask"
+                width={currentVolumeWidth}
+                height="24"
+              />
+            </defs>
+            <clipPath id="CurrentVolumeMask_1_">
```

Comment:
Will this be a problem if there is more than one? Is that the only way to assign the clip-path?

Reply:
I can use an instance counter.

Reply:
Added instance counter in https://github.com/digirati-co-uk/timeliner/pull/57/commits/948e9f35511fd64e40e1616af74146ea67989444